File: solutions/csharp/rna-transcription/1/RnaTranscription.cs
public static class RnaTranscription
{
    private static Dictionary<char, char> DnaToRna = new() {
        {'G', 'C'},
        {'C', 'G'},
        {'T', 'A'},
        {'A', 'U'}
    };
    
    public static string ToRna(string strand)
    {
        return string.Join("", strand.Select(c => DnaToRna[c]));
    }
}

File: solutions/python/rna-transcription/1/rna_transcription.py
dna_to_rna = {
    "G": "C",
    "C": "G",
    "T": "A",
    "A": "U"
}

def to_rna(dna_strand):
    result = [dna_to_rna[n] for n in dna_strand]
    return "".join(result)
